tests/instruments/test_chroma62024p6008.py

```diff
@@ -0,0 +1,12 @@
+from pymeasure.instruments.chroma.chroma62000p import Chroma62024p6008
```

Comment:
This file should go in a mirror directory to the source: `tests/instruments/chroma/test_chroma62000p.py`

So far we don't have good tests around instruments, since we don't have access to the instruments. We could do mock tests, but there is little value in my opinion to writing mocks without actually testing them at the instrument level. I would suggest that you put this test in a function, and behind a flag -- which we can set to ignore (e.g. `@pytest.mark.skip`). Happy to discuss this further, but I think this is beyond the scope of this PR -- I'm open to accepting it without the test file for now.
http://doc.pytest.org/en/latest/skipping.html#skipping-test-functions

Reply:
should I provide an example how to use it instead of test?

Reply:
> So far we don't have good tests around instruments, since we don't have access to the instruments. We could do mock tests, but there is little value in my opinion to writing mocks without actually testing them at the instrument level.

@cjermain I always found the instrumentkit way of [checking for the expected protocol](https://instrumentkit.readthedocs.io/en/latest/devguide/testing.html#expected-protocols) pretty neat - you can define sends and expected responses (as are often found in examples in the device manual), add the instrument code that executes the operation in question, and confirm that your defined instrument emits the correct commands.
What do you think? should that be added to pymeasure, too? maybe cf #99.